package.json

```diff
@@ -36,7 +37,7 @@
     "lodash-es": "^4.17.20",
     "sophia-bonding-curve": "github:aeternity/BondingCurve#1.0.0-alpha",
     "soundcloud-widget": "^0.2.1",
-    "tipping-contract": "github:aeternity/tipping-contract#v3.0.1-alpha1",
+    "tipping-contract": "github:aeternity/tipping-contract#v3.0.0-alpha1",
```

Comment:
why have you downgraded this package?

Reply:
I got that after merge with develop, will back v3.0.1-alpha1